src/stylesheets/components/_card.scss

```diff
@@ -17,12 +17,12 @@
     @include u-padding('105');
     @include u-bg('accent-cool-lighter');
 
-    a {
-      @include u-text('bold');
+    a, h2, h3 {
+      @include u-text('semibold');
       @include u-text('base-darker');
       @include u-text('no-underline');
       @include u-font-family('sans');
-      @include u-margin(0);
+      margin: 0 6px;
```

Comment:
Should this just be the 8px spacing now?

@include u-margin-y(0);
@include u-margin-x(2);